app/views/steps/shared/_relationship_step.html.erb

```diff
@@ -1,17 +1,15 @@
 <% heading = t('.heading', person_name: form_object.record.person.full_name, child_name:  form_object.record.minor.full_name) %>
 
-<h1 class="heading-xlarge gv-u-heading-xxlarge">
-  <%=  heading %>
-</h1>
+<h1 class="govuk-heading-xl"><%=  heading %></h1>
 
-<%= step_form form_object do |f| %>
-  <%=
-    f.radio_button_fieldset :relation, { legend_options: { class: 'visually-hidden', text: heading } } do |fieldset|
-      fieldset.radio_input Relation::MOTHER
-      fieldset.radio_input Relation::FATHER
-      fieldset.radio_input(Relation::OTHER) { f.text_field :relation_other_value }
-    end
-  %>
+<%= step_form form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+  <%= f.govuk_radio_buttons_fieldset :relation do %>
```

Comment:
Hey I think we can inject the heading into the fieldset legend, like this:

```ruby
f.govuk_radio_buttons_fieldset :relation, legend: { text: heading } do ...
```

Give it a try. That way we don't need the separate `<h1>` tag.